Week-5/5.3/index.js

```diff
@@ -0,0 +1,23 @@
+function hasDuplicate(arr) {
+    const mySet = new Set();
```

Comment:
@mr-rego We don't need to iterate over array to fill set, just by doing as new Set(...arr) will work out